api/base/urls.py

```diff
@@ -11,9 +11,14 @@
 
 urlpatterns = [
     ### API ###
-    url(base_pattern, include(patterns('',
-        url(r'^$', views.root),
-        url(r'^nodes/', include('api.nodes.urls', namespace='nodes')),
-        url(r'^users/', include('api.users.urls', namespace='users')),
-        url(r'^docs/', include('rest_framework_swagger.urls')),
-    )))] + static('/static/', document_root=settings.STATIC_ROOT)
\ No newline at end of file
+    url(base_pattern,
+        include(patterns('',
+                         url(r'^$', views.root),
```

Comment:
Hm, I preferred the style before this change, as it was more consistent with the rest of the codebase, but it's not a big deal.